content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -789,31 +789,39 @@ include metadata about the response.
 }
 ```
 
-The executed command is expected to print an `ExceCredential` to `stdout`. `k8s.io/client-go`
-will then use the returned bearer token in the `status` when authenticating against the
-Kubernetes API.
+When the plugin prints `ExecCredential` to `stdout`, the `k8s.io/cient-go` library
+looks for a bearer token or client TLS key and certificate (or all three) in the
+`status` field and uses it to authenticate against the Kubernetes API. The library can
+use a bearer token on its own (`token`), a client TLS key and certificate
+(`clientKeyData` and `clientCertificateData`; both must be present), or a combination
+of both methods. `clientCertificateData` may contain additional intermediate
+certificates to send to the server.
 
 ```json
 {
   "apiVersion": "client.authentication.k8s.io/v1alpha1",
   "kind": "ExecCredential",
   "status": {
-    "token": "my-bearer-token"
+    "token": "my-bearer-token",
+    "clientCertificateData": "-----BEGIN CERTIFICATE-----\n...\n-----END CERTIFICATE-----",
+    "clientKeyData": "-----BEGIN RSA PRIVATE KEY-----\n...\n-----END RSA PRIVATE KEY-----"
   }
 }
 ```
 
-Optionally, this output can include the expiry of the token formatted as a RFC3339 timestamp.
-If an expiry is omitted, the bearer token is cached until the server responds with a 401 HTTP
-status code. Note that this caching is only for the duration of process and therefore the plugin 
-is triggered each time the tool using the plugin is invoked.
+Optionally, this output can include the expiry of the token formatted as a
+RFC3339 timestamp. If an expiry is omitted, the bearer token and TLS
+credentials are cached until the server responds with a 401 HTTP status code
+or until the process completes.
```

Comment:
If expiry is present, cached credentials are still dropped when process exits.
This should probably be a separate sentence.